app/src/main/java/com/levonovyy/authentication/WelcomeActivity.java

```diff
@@ -0,0 +1,77 @@
+package com.levonovyy.authentication;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.widget.Button;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+
+public class WelcomeActivity extends AppCompatActivity {
+    private Button btnLogout;
+    private FirebaseAuth mFirebaseAuth;
```

Comment:
For you to know. The letter 'm' before the variable name means that it is a member of the object. According to Android naming conventions every field declared in the class should have this letter. And static variables should have 's' before the variable name. 
![image](https://user-images.githubusercontent.com/22887176/67406624-f71d6180-f5be-11e9-8863-1be552ce3c1f.png)